diff --git a/dist/client/config.d.ts b/dist/client/config.d.ts
index 2425167caa8b0f75217fe90db01527b1baad7b68..0e28ab8d2373947b0e44a820a22c9d0dd3b7d071 100644
--- a/dist/client/config.d.ts
+++ b/dist/client/config.d.ts
@@ -24,8 +24,8 @@ export declare const BLOCK_SCOUT_URLS: {
 };
 export declare const SUBGRAPH_VERSION = "0.22";
 export declare const SUBGRAPH_URLS: {
-    readonly mainnet: "https://subgraph.mainnet.fluence.dev/subgraphs/v0.22";
-    readonly testnet: "https://subgraph.testnet.fluence.dev/subgraphs/v0.22";
+    readonly mainnet: "https://subgraph-do.mainnet.fluence.dev/subgraphs/name/fluence-deal-contracts-bdd7ef17";
+    readonly testnet: "https://subgraph-do.testnet.fluence.dev/subgraphs/name/fluence-deal-contracts-bdd7ef17";
     readonly stage: "https://subgraph.stage.fluence.dev/subgraphs/v0.22";
     readonly local: "http://localhost:8000/subgraphs/name/fluence-deal-contracts";
 };
diff --git a/dist/client/config.js b/dist/client/config.js
index 81b322ce3436ea2d60935c67ac6ccb0c52657238..ac91d8f6da3bbc41c2a20090ef9603639983204f 100644
--- a/dist/client/config.js
+++ b/dist/client/config.js
@@ -18,8 +18,8 @@ export const BLOCK_SCOUT_URLS = {
 };
 export const SUBGRAPH_VERSION = "0.22";
 export const SUBGRAPH_URLS = {
-    mainnet: `https://subgraph.mainnet.fluence.dev/subgraphs/v${SUBGRAPH_VERSION}`,
-    testnet: `https://subgraph.testnet.fluence.dev/subgraphs/v${SUBGRAPH_VERSION}`,
+    mainnet: `https://subgraph-do.mainnet.fluence.dev/subgraphs/name/fluence-deal-contracts-bdd7ef17`,
+    testnet: `https://subgraph-do.testnet.fluence.dev/subgraphs/name/fluence-deal-contracts-bdd7ef17`,
     stage: `https://subgraph.stage.fluence.dev/subgraphs/v${SUBGRAPH_VERSION}`,
     local: "http://localhost:8000/subgraphs/name/fluence-deal-contracts",
 };
