diff --git a/dist/dealExplorerClient/indexerClient/queries/offers-query.generated.d.ts b/dist/dealExplorerClient/indexerClient/queries/offers-query.generated.d.ts
index 07fcf82e98fe2af73a8511447bf4beca4cb8a4c6..b73e6420afded231718a37242f5a9df56167372e 100644
--- a/dist/dealExplorerClient/indexerClient/queries/offers-query.generated.d.ts
+++ b/dist/dealExplorerClient/indexerClient/queries/offers-query.generated.d.ts
@@ -176,18 +176,7 @@ export type BasicPeerFragment = {
         __typename?: 'Provider';
         id: string;
     };
-    computeUnits?: Array<{
-        __typename?: 'ComputeUnit';
-        id: string;
-        worker?: {
-            __typename?: 'Worker';
-            id: string;
-        } | null;
-        provider: {
-            __typename?: 'Provider';
-            id: string;
-        };
-    }> | null;
+    computeUnitsTotal: number;
     currentCapacityCommitment?: {
         __typename?: 'CapacityCommitment';
         id: string;
diff --git a/dist/dealExplorerClient/indexerClient/queries/offers-query.generated.js b/dist/dealExplorerClient/indexerClient/queries/offers-query.generated.js
index 28a8fd8328169488b1bc7ba08e3f8a527927fa35..5c5daac9bfcd39f15b1b19bff33462568f301e64 100644
--- a/dist/dealExplorerClient/indexerClient/queries/offers-query.generated.js
+++ b/dist/dealExplorerClient/indexerClient/queries/offers-query.generated.js
@@ -65,9 +65,7 @@ export const BasicPeerFragmentDoc = gql `
   provider {
     id
   }
-  computeUnits(where: {deleted: false}) {
-    ...ComputeUnitBasic
-  }
+  computeUnitsTotal
   currentCapacityCommitment {
     ...ProviderBasicIdCapacityCommitment
   }
diff --git a/dist/dealExplorerClient/serializers/schemes.js b/dist/dealExplorerClient/serializers/schemes.js
index b4501d989701a715130598b8ee1a60a8f982fa87..99ec6fa4d0b317aca406f7c71917f59feb6d64b4 100644
--- a/dist/dealExplorerClient/serializers/schemes.js
+++ b/dist/dealExplorerClient/serializers/schemes.js
@@ -118,7 +118,7 @@ export function serializePeers(peers) {
     return peers.map((peer) => ({
         id: peerIdContractHexToBase58(peer.id),
         offerId: peer.offer.id,
-        computeUnits: serializeComputeUnitsFromBasic(peer.computeUnits ?? []),
+        computeUnitsTotal: peer.computeUnitsTotal,
         currentCapacityCommitment: peer.currentCapacityCommitment
             ? {
                 ...peer.currentCapacityCommitment,
diff --git a/dist/dealExplorerClient/types/schemes.d.ts b/dist/dealExplorerClient/types/schemes.d.ts
index a75092c0554124f166515bbbd6e232d40e5add74..5447abb8c7a4a306bec29c0a9865e70f4cc4417e 100644
--- a/dist/dealExplorerClient/types/schemes.d.ts
+++ b/dist/dealExplorerClient/types/schemes.d.ts
@@ -115,7 +115,7 @@ export interface Effector {
 export interface Peer {
     id: string;
     offerId: string;
-    computeUnits: ComputeUnit[];
+    computeUnitsTotal: number;
     currentCapacityCommitment?: {
         id: string;
         status: CapacityCommitmentStatus;
